src/components/navbar/Navbar.jsx

```diff
@@ -1,16 +1,17 @@
 import './Navbar.css';
 import { BrowserRouter as Router, Link, NavLink } from 'react-router-dom';
+import logo from './artparty-logo.png';
 
 const Navbar = () => {
     return (
-        <nav className="navbar">
-            <Router>
-                <Link to='/' >
-                    <img src='' alt="Art Party logo"/>
+        <Router>
+            <nav className="navbar">
+                <Link to='/' className="logo-link">
+                    <img src={ logo } alt="Art Party logo"/>
```

Comment:
let`s give `img` tag a `className` so we can use that in the CSS `maybe  something like `logo-Image` we just want to make sure that if another dev is looking at the `CSS` knows what we are referring.